src/cts/src/TreeBuilder.h

```diff
@@ -276,6 +280,8 @@ class TreeBuilder
   float aveArrival_ = 0.0;
   float topBufferDelay_ = 0.0;
   odb::dbInst* topBuffer_ = nullptr;
+  std::string topBufferName_;
```

Comment:
This feels like it should be storing a `dbInst*` and then converting to a string when necessary rather than the other way around.